test/integration/container_test.rb

```diff
@@ -10,9 +10,10 @@
       mount Back::Application,  at: '/back'
     end
 
-    @container = Hanami::Container.new
     Front::Application.load!
     Back::Application.load!
+
+    @container = Hanami::Container.new
```

Comment:
I delayed the creation of a `Hanami::Container` instance to simulate what happens on generated test suites. We can also just opt to initialize the container instance on tests that require it.